code/modules/events/event_container.dm

```diff
@@ -171,6 +171,7 @@ var/list/event_last_fired = list()
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Revenant", 				/datum/event/revenant, 					150),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Swarmer Spawn", 			/datum/event/spawn_swarmer, 			150, is_one_shot = 1),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Morph Spawn", 				/datum/event/spawn_morph, 				0, is_one_shot = 1),
+		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Traders",	 				/datum/event/traders,	 				150),
```

Comment:
This is still a moderate event, should be moved to major for reasons already discussed above.